webextension/background.js

```diff
@@ -9,10 +9,13 @@
 let gCurrentlyPromptingTab;
 
 const Config = (function() {
-  browser.experiments.aboutConfigPrefs.clearPrefsOnUninstall(["enabled"]);
+  browser.experiments.aboutConfigPrefs.clearPrefsOnUninstall(["enabled", "branch"]);
```

Comment:
Following https://github.com/mozilla/shield-studies-addon-template/blob/master/docs/TESTPLAN.md#install-the-add-on-and-enroll-in-the-study, let's call this `variation`, rather than `branch`.